src/toast/tests/runner.py

```diff
@@ -254,4 +256,9 @@ def test(name=None, verbosity=2):
     if ret > 0:
         sys.exit(ret)
 
+    alltimers = timing.gather_timers(comm=comm)
```

Comment:
We may be abusing the unit test system if we try to measure performance with them. Likewise for debugging. I learned just yesterday that it is utterly frustrating to debug parallel code with the unit tests when processes fail silently, only producing `error` on the terminal.

Reply:
I found it useful to compare timing in the unit tests for debugging.  But I will comment out the dumping for production use and just re-enable as needed.